tests/generators/OperationGenerator.ts

```diff
@@ -284,28 +291,62 @@ export default class OperationGenerator {
     return Buffer.from(JSON.stringify(operationJws));
   }
 
+  /**
+   * Creates a named anchored operation model from `OperationModel`.
+   */
+  public static createNamedAnchoredOperationModelFromOperationModel (
```

Comment:
This functions is not used here. Will it be used in later PR?

Reply:
It is being used in tests.